### ISSUE TITLE
notification off by default

### ISSUE THREAD

**Issue by cheenpo** [githubteacher/intermediate-may#2]
is there a way to have notifications be set to _off_ by default?

**Comment by cheenpo** [comment]
I guess maybe I meant that subscription to notifications was not on by default.

**Comment by githubstudent** [comment]
We'll cover notifications settings in Q&A in about 20 minutes.

**Comment by cheenpo** [comment]
did we cover this? just wanted to make sure I did not miss it

**Comment by githubstudent** [comment]
@cheenpo No, apologies! For the moment, please see [these docs](https://help.github.com/articles/configuring-notification-delivery-methods/).

**Comment by cheenpo** [comment]
perfect; thanks!

**Comment by githubteacher** [comment]
@cheenpo - I'm so sorry! Patrick asked me to cover these when we came back from break and I completely forgot. Were you able to find the info you needed? I am happy to connect with you to cover this if you still have questions!

**Comment by cheenpo** [comment]
yeap yeap.
that link helped.
well... maybe. I still get auto-added to any of the new issues popping up. how would i change that default?

I set unchecked this under the cog wheel:
Automatic watching
When you’re given push access to a repository, automatically receive notifications for it.

 Automatically watch repositories 

i bet... since I did not have that checked with today's when i first joined. might be why I am auto-subscribed to any new issue?
just trying to understand, not be anti-social. well, maybe a bit of both   ahahahaha
